app/models/round.rb

```diff
@@ -18,4 +28,8 @@ def winner
          .order('round_points DESC', 'number_of_correct_bets DESC', 'goal_difference ASC')
          .first
   end
+
+  def self.current_round_id
```

Comment:
```self``` isn't necessary here.

As I understand that correctly when there is no current round (like now, after group phase) the first round is displayed. I think that displaying the LAST round will be more logic there.